core/src/test/java/org/apache/calcite/sql/type/SqlTypeUtilTest.java

```diff
@@ -76,8 +76,6 @@ class SqlTypeUtilTest {
         is(true));
     assertThat(areSameFamily(ImmutableList.of(bigIntAndFloat, bigIntAndFloat)),
         is(true));
-    assertThat(areSameFamily(ImmutableList.of(bigIntAndFloat, bigIntAndFloat)),
```

Comment:
It should be `assertThat(areSameFamily(ImmutableList.of(floatAndBigInt, bigIntAndFloat)),
        is(true));`

1. [bigIntAndFloat, floatAndBigInt]
2. [bigIntAndFloat, bigIntAndFloat]
3. [floatAndBigInt, bigIntAndFloat]
4. [floatAndBigInt, floatAndBigInt]

Reply:
+1 agree with JiaJun.